src/registrar/admin.py

```diff
@@ -641,6 +663,18 @@ def formfield_for_manytomany(self, db_field, request, **kwargs):
             kwargs["queryset"] = models.Website.objects.all().order_by("website")  # Sort websites
         return super().formfield_for_manytomany(db_field, request, **kwargs)
 
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        # Removes invalid investigator options from the investigator dropdown
+        if db_field.name == "investigator":
+            kwargs["queryset"] = User.objects.filter(is_staff=True)
+            return db_field.formfield(**kwargs)
+        return super().formfield_for_foreignkey(db_field, request, **kwargs)
+
+    def get_queryset(self, request):
```

Comment:
This is throwing so many red flags. All columns should be sortable by default when you click on the column header. It seems that this functionality is working on some tables (domains) but not on others (DAs and DIs). Maybe we introduced a bug at some point?

In either case, I don't this this should be needed.